### PR TITLE
blog: add nx-plugin blog post

### PR DESCRIPTION
**Pull request by prabhatthapa**
We wanted to post the [Nx-plugins](https://mnscorp.sharepoint.com/sites/Software-Engineering/SitePages/Announcing-Nx-plugins-open-source-project.aspx) blog for the outer audience to make our presence in the open source community and attract talent to M&S Software Engineering community.
This PR will add the blog to our blog portal i.e https://engineering.marksandspencer.com/

### CODE REVIEW

File: _posts/2022-08-05-nx-plugins.md
Comment (aamir-mns) [inline _posts/2022-08-05-nx-plugins.md:16]: the path of this file is `assets/img/banners/terminal.png`. Does it just work by adding the file name like this?

**Review by aamir-mns — DISMISSED** [review]

File: _posts/2022-08-05-nx-plugins.md
Comment (prabhatthapa) [reply _posts/2022-08-05-nx-plugins.md:16]: Yeah, I guess so. It is a template configuration for the blog here and I followed the same pattern. Once we will merge then we can check if this works 👍

File: _posts/2022-08-05-nx-plugins.md
Comment (stevengonsalvez) [inline _posts/2022-08-05-nx-plugins.md:47]: just change it to "we" (as previous referred to as "we") , instead of "team within M&S"

File: _posts/2022-08-05-nx-plugins.md
Comment (stevengonsalvez) [inline _posts/2022-08-05-nx-plugins.md:49]: ^ same about the "team within M&S" - sounding like a separate reference.

**Review by stevengonsalvez — COMMENTED** [review]
just language comments , otherwise good !

File: _posts/2022-08-05-nx-plugins.md
Comment (prabhatthapa) [reply _posts/2022-08-05-nx-plugins.md]: Updated, please review again.

**Review by stevengonsalvez — APPROVED** [review]

**Pull request merged**
